server/index.js

```diff
@@ -131,6 +132,37 @@ app.post('/api/cart', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/orders', (req, res, next) => {
+
+  const { cartId } = req.session;
+
+  if (!(Number(cartId))) {
+    res.status(400).json({ err: 'There is no cartId' });
+  }
+  const name = req.body.name;
+  const creditCard = req.body.creditCard;
+  const shippingAddress = req.body.shippingAddress;
+
+  const sql = `
+    insert into "orders"
+    ("cartId", "name", "creditCard", "shippingAddress")
+    values($1, $2, $3, $4)
+    returning *
+    `;
+
+  const order = [cartId, name, creditCard, shippingAddress];
+
+  db.query(sql, order)
+    .then(result => {
+      delete result.rows[0].cartId;
```

Comment:
You want to `delete` the `cartId` from the `req.session`, not the data in the result.